.github/workflows/build.yml

```
@@ -1,27 +1,27 @@
-name: Build
-on:
-  workflow_dispatch:
-      inputs:
-        package_name:
-          description: "Package name of the game:"
-          required: true
+姓名:建立
+在……之上:
+workflow_dispatch:
+输入:
+package_name:
+描述:"com.tencent.tmgp.pubgmhd："
```

对照了一下所有被翻译的内容，正常情况下这可能是整个PR唯一变更了的行，只是添加一个包名
```suggestion
        description: "com.tencent.tmgp.pubgmhd"
```

---

鼠哥甚至帮他分析了一下，他真我哭

---

支持正义淘金

---

支持🙌

---

> 对照了一下所有被翻译的内容，正常情况下这可能是整个PR唯一变更了的行，只是添加一个包名

不太明白他加一个和平精英包名在这里干什么

---

But the package is PUBPMHD that is obviously not related with this repo. I thought this one appears here just because the wrong translation.

---

> > 对照了一下所有被翻译的内容，正常情况下这可能是整个PR唯一变更了的行，只是添加一个包名
> 
> 
> 
> 不太明白他加一个和平精英包名在这里干什么

See README, to use this repo need to add this line in action.

---

> 不太明白他加一个和平精英包名在这里干什么

https://github.com/Perfare/Zygisk-Il2CppDumper/blob/ede65d18a795882e37a2d988943d77a72d132c5d/README.zh-CN.md?plain=1#L7-L13

因为这个项目需要指定游戏软件的包名然后对这个包名对应的游戏的包跑处理

（还因为我刚刚才看到README是什么，原来这个是应该在fork了以后的本地分支做的，这个仓库是一个”模板“。那这位老哥的迷惑程度就更加一层了，你闲的没事来改模板仓库干嘛）

---

所以明明该根据自己需求fork这个仓库并且在他的分支里面加包名，却提了个pr到模板仓库还“汉化”，更难崩了

~~感觉像是玩和平精英想用模块作弊的结果连GitHub都没玩明白~~

---

试了一下，按readme说的到action里run的时候直接输入包名显然不会对代码有任何变更，更迷惑了。
game.h的那堆PR倒还算，理解

---

> 所以明明该根据自己需求fork这个仓库并且在他的分支里面加包名，却提了个pr到模板仓库还“汉化”，更难崩了
> 
> ~~感觉像是玩和平精英想用模块作弊的，结果连GitHub都没玩明白~~

难绷 想起之前lsp框架

---

> 所以明明该根据自己需求fork这个仓库并且在他的分支里面加包名，却提了个pr到模板仓库还“汉化”，更难崩了
> 
> ~感觉像是玩和平精英想用模块作弊的，结果连GitHub都没玩明白~

你是对的